components/collections/src/codepointinvliststringlist/mod.rs

```diff
@@ -0,0 +1,89 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+use alloc::string::{String, ToString};
+use crate::codepointinvlist::{CodePointInversionList, CodePointInversionListError};
+use displaydoc::Display;
+use zerovec::{VarZeroVec, ZeroVec};
+
+pub struct CodePointInversionListStringList<'data> {
+    cp_inv_list: CodePointInversionList<'data>,
+    str_list: VarZeroVec<'data, str>,
+}
+
+impl<'data> CodePointInversionListStringList<'data> {
+    pub fn try_from_lists(
```

Comment:
Please pair this with a function named `from_sorted_lists_unchecked`, with the same signature but not returning a `Result`, similar to https://icu4x.unicode.org/doc/litemap/struct.LiteMap.html#method.from_sorted_store_unchecked

Reply:
Done.